test/ibmq/test_account_client.py

```diff
@@ -248,6 +251,30 @@ def test_job_cancel(self):
                     # just retry.
                     self.assertIn('JOB_NOT_CANCELLED', str(ex))
 
+    def test_access_token_not_in_exception_traceback(self):
+        """Check that access token is replaced within chained request exceptions."""
+        backend = self.provider.backends.ibmq_qasm_simulator
+        circuit = transpile(self.qc1, backend, seed_transpiler=self.seed)
+        qobj = assemble(circuit, backend, shots=1)
+        api = backend._api
+
+        session_ = api.client_api.session
+        access_token_ = session_.access_token
```

Comment:
Any reason you don't just use `self.access_token`?

Reply:
I must have overseen that... Thanks for the catch!